xwiki-platform-core/xwiki-platform-store/xwiki-platform-store-filesystem-oldcore/src/test/java/org/xwiki/store/filesystem/internal/DefaultTemporaryAttachmentSessionsManagerTest.java

```diff
@@ -254,4 +256,20 @@ void removeUploadedAttachments()
         when(temporaryAttachmentSession.removeAttachments(documentReference)).thenReturn(true);
         assertTrue(this.attachmentManager.removeUploadedAttachments(documentReference));
     }
+
+    @Test
+    void temporarilyAttach() throws TemporaryAttachmentException
+    {
+        DocumentReference documentReference = mock(DocumentReference.class);
```

Comment:
Why a mock when an actual object could be easily instantiated?

Reply:
I always use mocks when I don't actually need an actual object in tests. That's how I see unit tests: anything not needed should be mocked as most as possible.